fiftyone/core/map/process.py

```diff
@@ -246,11 +253,11 @@ def _init_worker(
         process_sample_collection = dataset
 
     process_map_fcn = pickle.loads(map_fcn)
+    process_iter_fcn = pickle.loads(iter_fcn)
```

Comment:
I was going to comment on pickle and lambdas, but somehow passing a lambda to map_samples() works?? Not really sure how since lambdas should not be pickleable.

Reply:
`pickle` cannot serialize lambdas, which was a total deal-breaker for the entire `map_samples()` project.

Fortunately `dill` **can** serialize lambdas. Back in business!